maat-court-data-api/src/test/java/gov/uk/courtdata/assessment/mapper/FinancialAssessmentMapperTest.java

```diff
@@ -0,0 +1,28 @@
+package gov.uk.courtdata.assessment.mapper;
+
+import gov.uk.courtdata.builder.TestEntityDataBuilder;
+import gov.uk.courtdata.dto.IOJAssessorDetails;
+import gov.uk.courtdata.entity.FinancialAssessmentEntity;
+import gov.uk.courtdata.entity.UserEntity;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class FinancialAssessmentMapperTest {
+
+    private final FinancialAssessmentMapper financialAssessmentMapper = Mappers.getMapper(FinancialAssessmentMapper.class);
+
+    @Test
+    void shouldSuccessfullyCreateIOJAssessorDetails_fromUserEntity() {
+        FinancialAssessmentEntity financialAssessmentEntity = TestEntityDataBuilder.getFinancialAssessmentEntity();
+        final String userName = financialAssessmentEntity.getUserCreated();
+        UserEntity userEntity = TestEntityDataBuilder.getUserEntity(userName);
+        financialAssessmentEntity.setUserCreatedEntity(userEntity);
+
+        IOJAssessorDetails iojAssessorDetails = financialAssessmentMapper.createIOJAssessorDetails(financialAssessmentEntity);
```

Comment:
I wonder if perhaps the name of the method should be "shouldSuccessfullyCreateIOJAssessorDetails_fromFinancialAssessmentEntity" instead? Although the UserEntity is used within the process, the createIOJAssessorDetails method itself seems to take the financialAssessmentEntity.

Reply:
yes, well spotted, I'lll change that